src/@next/components/molecules/OrderTabel/OrderTable.tsx

```diff
@@ -0,0 +1,117 @@
+import React from "react";
+import { FormattedMessage, FormattedDate, useIntl } from "react-intl";
+import Media from "react-media";
+import { ThemeContext } from "styled-components";
+import Link from "next/link";
+
+import { TaxedMoney } from "@components/containers";
+import { paths } from "@paths";
+import { commonMessages, translateOrderStatus } from "@temp/intl";
+
+import { Thumbnail } from "..";
+import { generateProductUrl } from "../../../../core/utils";
+
+import * as S from "./styles";
+import { IProps } from "./types";
+
+const header = (matches: boolean) => (
+  <S.HeaderRow>
+    <S.IndexNumber>
+      <FormattedMessage defaultMessage="Index Number" />
+    </S.IndexNumber>
+    {matches && (
+      <>
+        <S.ProductsOrdered>
+          <FormattedMessage defaultMessage="Products Ordered" />
+        </S.ProductsOrdered>
+        <S.DateOfOrder>
+          <FormattedMessage defaultMessage="Date of Order" />
+        </S.DateOfOrder>
+        <S.Value>
+          <FormattedMessage defaultMessage="Value" />
+        </S.Value>
+      </>
+    )}
+    <S.Status>
+      <FormattedMessage {...commonMessages.status} />
+    </S.Status>
+  </S.HeaderRow>
+);
+
+export const OrderTable: React.FC<IProps> = ({ orders, isGuest }: IProps) => {
+  const theme = React.useContext(ThemeContext);
+  const intl = useIntl();
+
+  return (
+    <S.Wrapper>
+      <Media
+        query={{
+          minWidth: theme.breakpoints.largeScreen,
+        }}
+      >
+        {(matches: boolean) => {
+          return (
+            <>
+              <S.Row>{header(matches)}</S.Row>
+              {orders &&
+                orders.map(order => {
+                  const date = new Date(order.node.created);
+                  return (
+                    <Link
+                      href={{
+                        pathname: isGuest
+                          ? paths.guestOrderDetail
+                          : paths.accountOrderDetail,
+                        query: { token: order.node.token },
+                      }}
+                      key={order.node.number}
+                    >
```

Comment:
Renamed it coz of typo and added ^ Link.

Reply:
The directory is still called `OrderTabel` 😄

Reply:
I was sure I renemd it as well... 🙈